src/MCPClient/lib/archivematicaClient.py

```diff
@@ -161,14 +127,17 @@ def executeCommand(gearman_worker, gearman_job):
 
         # Add useful environment vars for client scripts
         env_updates = {
```

Comment:
This is the environment of the client scripts. I modify the `PYTHONPATH` to make sure that the client scripts can find the settings package of MCPClient.